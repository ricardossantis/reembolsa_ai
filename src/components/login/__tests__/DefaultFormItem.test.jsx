import DefaultFormItem from "../default-form-item/index";
import React from "react";
import renderer from "react-test-renderer";

describe("Render Test", () => {
  it("renders DefaultFormItem", () => {
    const component = renderer.create(<DefaultFormItem/>).toJSON();
    expect(component).toMatchSnapshot();
  });
});

