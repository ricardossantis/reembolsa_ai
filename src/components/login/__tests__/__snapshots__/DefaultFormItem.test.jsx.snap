// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders DefaultForm 1`] = `
<div
  className="ant-row ant-form-item sc-bdnylx itAIRD"
  style={Object {}}
>
  <div
    className="ant-col ant-form-item-control"
    style={Object {}}
  >
    <div
      className="ant-form-item-control-input"
    >
      <div
        className="ant-form-item-control-input-content"
      />
    </div>
  </div>
</div>
`;

exports[`Render Test renders DefaultFormItem 1`] = `
<div
  className="ant-row ant-form-item sc-bdnylx itAIRD"
  style={Object {}}
>
  <div
    className="ant-col ant-form-item-control"
    style={Object {}}
  >
    <div
      className="ant-form-item-control-input"
    >
      <div
        className="ant-form-item-control-input-content"
      />
    </div>
  </div>
</div>
`;
